configure.ac

```diff
@@ -103,6 +103,10 @@ for MODULE in lxml polib; do
     fi
 done
 
+AC_CHECK_HEADERS([sys/random.h],
+                [],
+                [AC_MSG_NOTICE([cannot find sys/random.h])])
+
```

Comment:
I added a check for sys/random.h header. we can use `#ifdef HAVE_SYS_RANDOM_H` directive.